modules/apps/asset/asset-list-web/npmscripts.config.js

```diff
@@ -12,11 +12,8 @@
  * details.
  */
 
-const preset = require('@liferay/npm-scripts/src/presets/standard');
-
 module.exports = {
 	build: {
-		dependencies: [...preset.build.dependencies, 'asset-taglib'],
```

Comment:
As I said in a commit that has been lost by GitHub somewhere :roll_eyes: ...

This is the only type of override we do over the default presets: adding dependencies. If we used the array merge options we would have to copy the whole default array here which is something we may probably not want.

Reply:
It's not all-or-nothing though. Look at what we do with [the `babelMerge` strategy](https://github.com/liferay/liferay-frontend-projects/blob/master/projects/npm-tools/packages/npm-scripts/src/utils/deepMerge.js#L144-L147). If the only thing in here that needs "special handling" is `build.dependencies`, we can probably make that happen, especially if what you said in that other commit eaten by GitHub is true (that we only ever add, never remove).